 Remove throttle logcat for bluetooth

Due to reduced buffer sizes for bluetooth it is possible to throttle
regularly, do not logcat throttle for bluetooth to avoid spamming.

diff --git a/services/audioflinger/Threads.cpp b/services/audioflinger/Threads.cpp
index 77b9799..6ef9b3a 100644
--- a/services/audioflinger/Threads.cpp
+++ b/services/audioflinger/Threads.cpp
@@ -3070,7 +3070,9 @@ bool AudioFlinger::PlaybackThread::threadLoop()
                             uint32_t diff = mThreadThrottleTimeMs - mThreadThrottleEndMs;
                             if (diff > 0) {
                                 // notify of throttle end on debug log
-                                ALOGD("mixer(%p) throttle end: throttle time(%u)", this, diff);
+                                // but prevent spamming for bluetooth
+                                ALOGD_IF(!audio_is_a2dp_out_device(outDevice()),
+                                        "mixer(%p) throttle end: throttle time(%u)", this, diff);
                                 mThreadThrottleEndMs = mThreadThrottleTimeMs;
                             }
                         }
